programs/conditional_vault/src/instructions/resolve_question.rs

```diff
@@ -0,0 +1,33 @@
+use super::*;
+
+#[derive(AnchorSerialize, AnchorDeserialize)]
+pub struct ResolveQuestionArgs {
+    pub payout_numerators: Vec<u32>,
+}
+
+#[derive(Accounts)]
+#[instruction(args: ResolveQuestionArgs)]
+pub struct ResolveQuestion<'info> {
+    #[account(mut, has_one = oracle)]
+    pub question: Account<'info, Question>,
+    pub oracle: Signer<'info>,
+}
+
+impl ResolveQuestion<'_> {
+    pub fn handle(ctx: Context<Self>, args: ResolveQuestionArgs) -> Result<()> {
+        let question = &mut ctx.accounts.question;
+
+        require_eq!(
+            args.payout_numerators.len(),
+            question.num_outcomes(),
+            VaultError::InvalidNumPayoutNumerators
+        );
+
```

Comment:
Should we check that the options add up to 100%:

        require_eq!(
            question.payout_denominator,
            args.payout_numerators.iter().sum(),
            VaultError::InvalidNumPayoutNumerators
        );

Reply:
this is implicit because we're setting question.payout_denominator equal to this sum, so check not needed